src/panels/lovelace/editor/hui-element-editor.ts

```diff
@@ -280,6 +289,14 @@ export abstract class HuiElementEditor<T, C = any> extends LitElement {
     if (this._configElement && changedProperties.has("context")) {
       this._configElement.context = this.context;
     }
+
+    if (
+      this._configElement &&
+      this._configElement.setClipboard &&
+      this.getClipboard()
+    ) {
+      this._configElement.setClipboard?.(this.getClipboard());
```

Comment:
This kind of feels wrong here, like maybe it should be in `hui-card-element-editor` instead of `hui-element-editor`, but I didn't find any way to make that work, since the `_configElement` is only accessible here.